frontend/src/main/scala/bloop/data/BuildSettings.scala

```diff
@@ -0,0 +1,70 @@
+package bloop.data
+
+import bloop.engine.BuildLoader
+import bloop.logging.Logger
+import bloop.logging.DebugFilter
+import java.nio.charset.StandardCharsets
+import bloop.config.ConfigEncoderDecoders
+import io.circe.Printer
+import io.circe.Json
+import io.circe.parser
+import bloop.io.AbsolutePath
+import java.nio.file.Files
+import io.circe.JsonObject
+import bloop.DependencyResolution
+import scala.util.Try
+import io.circe.ObjectEncoder
+import io.circe.Decoder
+import io.circe.derivation._
+import java.nio.file.Path
+import scala.util.Failure
+import scala.util.Success
+
+case class BuildSettings(semanticDBVersion: Option[String])
+
+object BuildSettings {
+
+  /** File to store Metals specific settings*/
```

Comment:
Maybe we could call the var `metalsSettingsFileName` and avoid the comment?

Reply:
This might be needed for something more - I thinkw e can keep this name